list/PageableList.js

```diff
@@ -315,8 +221,87 @@ define([
 			};
 		}),
 
+		/*jshint maxcomplexity:14*/
+		computeProperties: function (props) {
+			if (this.pageLength > 0) {
+				if ("_busy" in props || "hideOnPageLoad" in props || "autoPaging" in props || "showNoItems" in props) {
+					this._updateListView();
+				}
+
+				if ("loadPreviousMessage" in props || "loadingMessage" in props || "_showPreviousPageLoader" in props) {
+					if (this.previousPageLoader && this.previousPageLoader.deliver) {
+						this.previousPageLoader.labels = {
+							loadMessage: string.substitute(this.loadPreviousMessage, this),
+							loadingMessage: this.loadingMessage
+						};
+						this.previousPageLoader.deliver();
+					}
+				}
+
+				if ("loadNextMessage" in props || "loadingMessage" in props || "_showNextPageLoader" in props) {
+					if (this.nextPageLoader && this.nextPageLoader.deliver) {
+						this.nextPageLoader.labels = {
+							loadMessage: string.substitute(this.loadNextMessage, this),
+							loadingMessage: this.loadingMessage
+						};
+						this.nextPageLoader.deliver();
+					}
+				}
+			}
+		},
+
 		//////////// Private methods ///////////////////////////////////////
 
+		/*
+		 * Ovveride deliteful/List#_updateListview to update the widget view.
```

Comment:
Oh and also, it's marked as private in both List and PageableList but if it were really private it wouldn't be in PageableList.js at all.   So should be marked as protected, and not have an underscore.